client/surface/surface.js

```diff
@@ -14,14 +14,13 @@ limitations under the License.
 ==============================================================================*/
 
 define(function(require) {
-  'use strict';
-  var Rectangle = require('lib/rectangle');
-  var geometry = require('lib/geometry');
-  var info = require('client/util/info');
+  const Rectangle = require('lib/rectangle');
+  const geometry = require('lib/geometry');
+  const info = require('client/util/info');
 
   // Installs a surface in the given container & readies the surface for
   // drawing.
-  var Surface = function Surface(container, wallGeometry) {
+  const Surface = function Surface(container, wallGeometry) {
```

Comment:
It seems we could make this a class (http://es6-features.org/#ClassDefinition). It would probably make it easier to read.